docs/_static/flow-oidc.diag

```diff
@@ -0,0 +1,22 @@
+seqdiag {
+    browser -> app;
+    browser <- app [label = "redirect to /auth/openid/login"];
+    browser -> gafaelfawr [label = "/auth/openid/login"];
+    browser <- gafaelfawr [label = "redirect to /login"]
+    browser -> gafaelfawr [label = "/login"];
+    browser <- gafaelfawr [label = "redirect to IdP"];
+    browser -> IdP [label = "authenticate"];
+    browser <- IdP [label = "redirect to /login"];
+    browser -> gafaelfawr [label = "/login"];
+               gafaelfawr -> IdP [label = "get data"];
+               gafaelfawr <- IdP [label = "user data"];
+    browser <- gafaelfawr [label = "redirect to /auth/openid/login"];
+    browser -> gafaelfawr [label = "/auth/openid/login"];
+    browser <- gafaelfawr [label = "redirect to app"];
```

Comment:
is there a cookie in there or something?  This is super helpful though

Reply:
Yeah, `/login` sets a cookie. I have now made that more explicit. The redirect back from the app includes the code in the URL as a GET parameter.